src/components/Inputs/Input.tsx

```diff
@@ -0,0 +1,37 @@
+import { forwardRef, type DetailedHTMLProps, type InputHTMLAttributes, type ReactNode } from "react";
+import { twJoin } from "tailwind-merge";
+import { Loader } from "../Loader";
+import "./Input.css";
+
+export interface InputProps
+  extends Omit<DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>, "prefix" | "suffix"> {
+  className?: string;
+  wrapperClassName?: string;
+  suffix?: ReactNode;
+  isLoading?: boolean;
+  onSuffixClick?: () => void;
```

Comment:
```suggestion
```
No need for such props in this component. You can extend it inside dApp by creating SearchInput component

Reply:
We need add disabled, error, warning, success states